docs/source/usage.rst

```diff
@@ -214,6 +226,15 @@ Example:
   1 snp_c 65429279
   1 snp_d 107421759
 
+Alternative Allele Frequency File
+=================================
+
+The alternative allele frequency file allows for user-defined population alternative allele frequency or frequencies where metafounders are used. The alternative allele frequency file contains the metafounder group denoted MF_x, where x is a numerical value, and the associated alternative allele frequency. If any alternative allele frequencies are missing from the file, the default of 0.5 will be used for these metafounders.
+
+Example:
+::
+  MF_1 0.3
```

Comment:
Provide one example with just one MF, MF_1, and more loci - as many as we have in the geno file.

Then another example with two MF, again with as many markers ...